tests/Providers/Trustpilot/Request/AccessTokenRequestTest.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace PodPoint\Reviews\Tests\Providers\Trustpilot\Request;
+
+use PodPoint\Reviews\AccessToken;
+use PodPoint\Reviews\Providers\Trustpilot\Request\AccessTokenRequest;
+use PodPoint\Reviews\Tests\TestCase;
+
+/**
+ * Class AccessTokenRequestTest
+ * @package PodPoint\Reviews\Tests\Providers\Trustpilot\Request
+ */
+class AccessTokenRequestTest extends TestCase
+{
+    protected $requiredFields;
+    protected $options;
+
+    protected function setUp()
+    {
+        $this->requiredFields = [
+            'apiKey',
+            'apiSecret',
+            'username',
+            'password'
+        ];
+
+        $this->options = [
+            'apiKey' => 'api-key',
+            'apiSecret' => 'api-secret-123',
+            'username' => 'api-user',
+            'password' => 'api-password',
+        ];
+    }
+
+    /**
+     * Test construct to make sure properties are set.
+     *
+     * @throws \PodPoint\Reviews\Exceptions\ValidationException
+     */
+    public function testConstruct()
+    {
+        $mockedApiClient = $this->getMockedApiClient();
+        $request = new AccessTokenRequest($mockedApiClient, $this->options);
+
+        $this->assertEquals($mockedApiClient, $request->getHttpClient());
+        $this->assertEquals($this->options, $request->getOptions());
+    }
+
+    /**
+     *
+     */
+    public function testRequiredFields()
+    {
+        $mockedApiClient = $this->getMockedApiClient();
+        $request = new AccessTokenRequest($mockedApiClient, $this->options);
+
+        $this->assertEquals($this->requiredFields, $request->requiredFields());
+    }
+
+    /**
+     * Making sure the Request instance is build as expected.
+     */
+    public function testGetRequest()
+    {
+        $mockedApiClient = $this->getMockedApiClient();
+        $serviceReviewRequest = new AccessTokenRequest($mockedApiClient, $this->options);
+
+        $request = $serviceReviewRequest->getRequest();
+
+        $this->assertInstanceOf(\Psr\Http\Message\RequestInterface::class, $request);
+
+        $this->assertEquals('https', $request->getUri()->getScheme());
+        $this->assertEquals('api.trustpilot.com', $request->getUri()->getHost());
+        $this->assertEquals('/v1/oauth/oauth-business-users-for-applications/accesstoken', $request->getUri()->getPath());
+        $this->assertEquals('', $request->getUri()->getQuery());
+    }
+
+    /**
+     * Send should return an array by converting the json response.
+     */
```

Comment:
I think this could throw an exception. Not 100% sure but if so can you please add to test comment also for consistency